evmbin/src/main.rs

```diff
@@ -405,15 +457,118 @@ mod tests {
 			"--only=add11",
 			"--json",
 			"--std-json",
-			"--std-dump-json"
+			"--std-dump-json",
+			"--std-out-only",
+			"--std-err-only",
 		]);
 
 		assert_eq!(args.cmd_state_test, true);
 		assert!(args.arg_file.is_some());
+		assert_eq!(args.flag_chain, Some("homestead".to_owned()));
+		assert_eq!(args.flag_only, Some("add11".to_owned()));
 		assert_eq!(args.flag_json, true);
 		assert_eq!(args.flag_std_json, true);
 		assert_eq!(args.flag_std_dump_json, true);
-		assert_eq!(args.flag_chain, Some("homestead".to_owned()));
-		assert_eq!(args.flag_only, Some("add11".to_owned()));
+		assert_eq!(args.flag_std_out_only, true);
+		assert_eq!(args.flag_std_err_only, true);
 	}
+
+	#[test]
+	fn should_parse_state_test_command_from_state_test_json_file() {
+		let s = r#"{
```

Comment:
nit: putting this long string into a file and use `include_str!` is probably better.

Reply:
Thanks. I've pushed commit cd742a6245f96eff4988cf7762b47f9ed76fdcb9 that uses `include_str!` and I've updated the tests